contents/docs/libraries/ios/index.mdx

```diff
@@ -36,6 +36,102 @@ PostHog supports the following Apple platforms:
 
 <IOSSendEvents />
 
+## Autocapture 
+
+PostHog autocapture automatically tracks the following events for you:
+
+-   **Application Opened** - when the app is opened from a closed state or when the app comes to the foreground (e.g. from the app switcher)
+-   **Application Backgrounded** - when the app is sent to the background by the user
+-   **Application Installed** - when the app is installed.
+-   **Application Updated** - when the app is updated.
+-   **$screen** - when the user navigates (if using `UIViewController`)
+-   **$autocapture** - interactions with elements in a screen (if using `UIKit`)
+
+> 🚧 **Note:** `$autocapture` is currently supported only in UIKit and remains in an experimental state, with the feature disabled by default. SwiftUI support is planned for an upcoming release.
+
+### Autocapture configuration
+
+You can enable or disable Autocapture through the `PostHogConfig` object.
+
+```swift
+let POSTHOG_API_KEY = "<ph_project_api_key>"
+// usually 'https://us.i.posthog.com' or 'https://eu.i.posthog.com'
+let POSTHOG_HOST = "https://us.i.posthog.com"
+
+let config = PostHogConfig(apiKey: POSTHOG_API_KEY, host: POSTHOG_HOST)
+
+// capture screen views
+config.captureScreenViews = true 
+// capture application lifecycle events
+config.captureApplicationLifecycleEvents = true
+// enable $autocapture events
+config.autocapture = true // TIP: This is false by default
+// for $autocapture events, capture text input changes and edits
+config.autocaptureConfig.captureTextEdits = true 
+// for $autocapture events, capture gestures such as swipes and taps
+config.autocaptureConfig.captureGestures = true
+// for $autocapture events, capture control actions such as button presses
+config.autocaptureConfig.captureControlActions = true
```

Comment:
I am not convinced control actions are clear for users, the comment helps but is there a better name for this?